src/Raven.Client/Documents/Session/IDocumentSession.TimeSeriesFor.cs

```diff
@@ -12,15 +12,34 @@ namespace Raven.Client.Documents.Session
     /// </summary>
     public partial interface IDocumentSession
     {
+        /// <inheritdoc cref="IAsyncDocumentSession.TimeSeriesFor"/>
         ISessionDocumentTimeSeries TimeSeriesFor(string documentId, string name);
+
+        /// <inheritdoc cref="IAsyncDocumentSession.TimeSeriesFor"/>
         ISessionDocumentTimeSeries TimeSeriesFor(object entity, string name);
+
+        /// <inheritdoc cref="IAsyncDocumentSession.TimeSeriesFor{TValues}"/>
         ISessionDocumentTypedTimeSeries<TValues> TimeSeriesFor<TValues>(object entity, string name = null) where TValues : new();
+
+        /// <inheritdoc cref="IAsyncDocumentSession.TimeSeriesFor{TValues}"/>
         ISessionDocumentTypedTimeSeries<TValues> TimeSeriesFor<TValues>(string documentId, string name = null) where TValues : new();
+
+        /// <inheritdoc cref="IAsyncDocumentSession.TimeSeriesFor{TValues}"/>
         ISessionDocumentRollupTypedTimeSeries<TValues> TimeSeriesRollupFor<TValues>(object entity, string policy, string raw = null) where TValues : new();
+
+        /// <inheritdoc cref="IAsyncDocumentSession.TimeSeriesFor{TValues}"/>
         ISessionDocumentRollupTypedTimeSeries<TValues> TimeSeriesRollupFor<TValues>(string documentId, string policy, string raw = null) where TValues : new();
+
+        /// <inheritdoc cref="IAsyncDocumentSession.TimeSeriesFor"/>
```

Comment:
Shouldn't it inheric docs from IncrementalTimeSeriesFor?

Reply:
done